### DISCUSSION TITLE
[Script request] InvenTree

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#622] [Request script]
### Application Name

InvenTree

### Website

https://github.com/inventree/InvenTree

### Description

InvenTree is an open-source Inventory Management System which provides powerful low-level stock control and part tracking. The core of the InvenTree system is a Python/Django database backend which provides an admin interface (web-based) and a REST API for interaction with external interfaces and applications. A powerful plugin system provides support for custom applications and extensions.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Narkowii** [comment]
nice app

**Comment by MickLesk** [comment]
Look at this:
https://github.com/community-scripts/ProxmoxVED/issues/11

can be tested, please give feedback into this link. After sucessfully test, it will be merged against main branch.

**Comment by MickLesk** [comment]
merged
